frontend/src/app/pages/exercises/exercise/shared/emergency-operations-center/send-alarm-group-interface/send-alarm-group-interface.component.html

```diff
@@ -1,72 +1,187 @@
+
+    <div *ngIf="alarmGroups$ | async | values as alarmGroups" class="card-body">
         <div
             *appLet="
                 transferPoints$
                     | async
                     | values
                     | orderBy : getTransferPointOrderByValue as transferPoints
             "
-            ngbDropdown
-            class="d-inline-block float-end"
+            class="d-flex flex-row justify-content-end"
         >
-            Ziel:
-            <button
-                ngbDropdownToggle
-                type="button"
-                class="btn btn-outline-primary btn-sm ms-3"
-                data-cy="sendAlarmGroupChooseTargetButton"
-                [disabled]="transferPoints.length === 0"
+            <div
+                ngbDropdown
+                class="input-group input-group-sm"
+                style="max-width: 300px; width: unset"
             >
-                <app-transfer-point-name
-                    *ngIf="targetTransferPointId; else noTargetTransferPoint"
-                    [transferPointId]="targetTransferPointId"
+                <span class="input-group-text">Ziel:</span>
+                <button
+                    ngbDropdownToggle
+                    type="button"
+                    class="btn btn-outline-primary btn-sm"
+                    data-cy="sendAlarmGroupChooseTargetButton"
+                    [disabled]="transferPoints.length === 0"
                 >
-                </app-transfer-point-name>
-                <ng-template #noTargetTransferPoint>
-                    <span class="text-muted"> Kein Ziel ausgewählt </span>
-                </ng-template>
-            </button>
-            <div ngbDropdownMenu>
+                    <app-transfer-point-name
+                        *ngIf="
+                            targetTransferPointId;
+                            else noTargetTransferPoint
+                        "
+                        [transferPointId]="targetTransferPointId"
+                    >
+                    </app-transfer-point-name>
+                    <ng-template #noTargetTransferPoint>
+                        <span class="text-muted"> Kein Ziel ausgewählt </span>
+                    </ng-template>
+                </button>
+                <div ngbDropdownMenu>
+                    <button
+                        ngbDropdownItem
+                        (click)="targetTransferPointId = undefined"
+                        type="button"
+                        class="btn btn-primary"
+                        *ngIf="targetTransferPointId !== undefined"
+                    >
+                        Kein Ziel
+                    </button>
+                    <ng-container
+                        *ngFor="
+                            let transferPoint of transferPoints;
+                            trackBy: 'id' | appTrackByProperty : transferPoints
+                        "
+                    >
```

Comment:
Why don't you just apply the `*ngFor` on the `<button>` element?

Reply:
Because you cannot have two ng-star directives on the same html element